### ISSUE TITLE
Image scroll with info overlay [BUGFIX]

### ISSUE THREAD

**Issue by mowgli** [BestImageViewer/geeqie#355]
Info or Histogram overlay may leave ugly traces as the image is scrolled. This patch should fix the problem.

Reported by: unqwist

**Comment by mowgli** [comment]
Original patch:
```patch
diff -ur geeqie-1.0.orig/src/pixbuf-renderer.c geeqie-1.0/src/pixbuf-renderer.c
--- geeqie-1.0/src/pixbuf-renderer.c	2010-02-17 21:21:19.000000000 +0000
+++ geeqie-1.0/src/pixbuf-renderer.c	2012-07-08 15:44:21.000000000 +0000
@@ -3593,6 +3593,8 @@
 
 	if (!pr->pixbuf && !pr->source_tiles_enabled) return;
 
+	pr_overlay_queue_all(pr);
+
 	old_x = pr->x_scroll;
 	old_y = pr->y_scroll;
 
@@ -3659,11 +3661,6 @@
 				  x1 + pr->x_offset, y1 + pr->y_offset, w, h);
 		g_object_unref(gc);
 
-		if (pr->overlay_list)
-			{
-			pr_overlay_queue_all(pr);
-			}
-
 		w = pr->vis_width - w;
 		h = pr->vis_height - h;
 
```

**Comment by mowgli** [comment]
Hmm, seems to be fixed already with a1751b3.
